stdlib/3/time.pyi

```diff
@@ -65,3 +73,18 @@ def strptime(string: str,
              format: str = ...) -> struct_time: ...
 def time() -> float: ...
 def tzset() -> None: ...  # Unix only
+
+if sys.version_info >= (3, 3):
+    def clock_getres(int) -> float: ...
+
```

Comment:
The blank lines between function stubs match the rest of the file (see the lines above these). I can remove the blanks here, but do you want me to also remove the blank lines elsewhere in the file?